### ISSUE TITLE
Script.execute on OutputStream

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#140]
```
Problem: org.h2.tools.Script.execute only works on a File-path (String) but
I would let to have it work on a generic java.io.OutputStream.

-> I'd like to have an API
 public static void execute(String url, String user, String password,
OutputStream os) throws SQLException { ... }

Thanks and keep up the good work!
```

Original issue reported on code.google.com by `synp...@hotmail.com` on 16 Nov 2009 at 4:33

**Comment by GoogleCodeExporter** [comment]
```
It seems to be easy to implement. I will try for then next release.
```

Original comment by `thomas.t...@gmail.com` on 20 Nov 2009 at 9:43
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in version 1.2.124
```

Original comment by `thomas.t...@gmail.com` on 20 Nov 2009 at 4:55
- Changed state: **Fixed**
